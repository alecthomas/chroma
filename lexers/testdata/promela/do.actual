init {
  a ? b -> a ! b;
}
