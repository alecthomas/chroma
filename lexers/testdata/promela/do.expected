[
  {"type":"KeywordDeclaration","value":"init"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"{"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"Name","value":"a"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Operator","value":"?"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Name","value":"b"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Operator","value":"-\u003e"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Name","value":"a"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Operator","value":"!"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Name","value":"b"},
  {"type":"Punctuation","value":";"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"Punctuation","value":"}"},
  {"type":"TextWhitespace","value":"\n"}
]
